docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -3,53 +3,54 @@
+This Quick Start guide presents a very basic example using automatic discovery to provision nodes through SNMP.
 
-* Use plugins to integrate with external systems.
-* Specify more complex entity detection with OpenNMS detectors (ActiveMQ, DNS, FTP, JDBC, TCP, and so on).
-* Create policies to manage provisioning behavior.
+== SNMP in {page-component-title}
+
+{page-component-title} uses SNMP to understand network and node topology, collect performance data, and enable SNMP service monitoring of nodes.
+SNMP service monitoring ensures that SNMP is available on a given node.
+It checks a configurable OID value to check if it returns a specified value.
+Tracking the state of this value can be used to trigger a notification if it represents an error or another condition that may need attention.
 
-Refer to xref:operation:deep-dive/provisioning/introduction.adoc[Provisioning] in the Deep Dive section for details about these and other provisioning-related tasks.
+The service also gathers basic information about a monitored device, including the device type, its name, its location, and so on.
 
 [[provision-snmp-configuration]]
 == Configure SNMP for provisioning
 
 To collect data through SNMP or monitor the availability of SNMP services on a node, you need to configure SNMP for provisioning.
 This ensures that {page-component-title} can immediately scan newly discovered devices for entities.
-
-Proper SNMP configuration lets {page-component-title} understand network and node topology and automatically enables performance data collection.
-{page-component-title} updates network topology as it provisions nodes.
+{page-component-title} updates the network topology--the visual representation of the monitored network and its components--as it provisions nodes.
 
 This procedure illustrates a very basic SNMP configuration using the default parameters.
-You can customize these values for your own network needs.
+You can customize these values for your own network needs, although the default settings are sufficient for a basic {page-component-title} setup:
 
-. Click the *Gear* symbol in the top-right of the screen.
+. Click the *gear* symbol in the top-right of the screen.
 . Under Provisioning, click *Configure SNMP Community Names by IP Address*.
-. Under General Parameters, specify an IP range using the *First IP Address* and *Last IP Address* boxes, and accept the default values for the other fields.
-. Scroll to the bottom and click *Save Config*.
+. Configure the community names as appropriate for your environment:
+** Enter a valid IP address in the *IP Address* box, and click *Look Up*.
+** Choose an SNMP version (`v2c` or `v1`) from the *Version* list.
++
+NOTE: If you are using SNMPv3, you need to configure details for a User-Based Security Model (USM) user instead of a community string.
+{page-component-title} supports SNMPv3 and USM, but these topics are outside of the scope of this guide.
```

Comment:
```suggestion
{page-component-title} supports SNMPv3 and USM, but these topics are outside of the scope of this Quick Start guide.
```
Just to make it clear we don't mean the core docs. We will document this soon!

Reply:
I'll note this for cross-reference once we document it. :)